test/ctia/domain/access_control_test.clj

```diff
@@ -83,110 +84,126 @@
-       (test-authorized_groups-mismatch "green" sut/allow-read? false?)
-       (test-authorized_users-match "green" sut/allow-read? true?)
-       (test-authorized_groups-match "green" sut/allow-read? true?))))
+   (fn [get-in-config]
+    (let [allow-read? #(sut/allow-read? %1 %2 get-in-config)]
+     (testing "green TLPs should disallow document read to everyone"
+       (test-matching-user "green" allow-read? true?)
+       (test-matching-group "green" allow-read? true?)
+       (test-no-group "green" allow-read? false?)
+       (test-user-group-mismatch "green" allow-read? false?)
+       (test-authorized_users-mismatch "green" allow-read? false?)
+       (test-authorized_groups-mismatch "green" allow-read? false?)
+       (test-authorized_users-match "green" allow-read? true?)
+       (test-authorized_groups-match "green" allow-read? true?))))))
 
 (deftest allow-read?-tlp-amber-max-record-visibility-group-test
   (with-max-record-visibility-group
-    #(testing "amber TLPs should allow document read to same group"
-       (test-matching-user "amber" sut/allow-read? true?)
-       (test-matching-group "amber" sut/allow-read? true?)
-       (test-no-group "amber" sut/allow-read? false?)
-       (test-user-group-mismatch "amber" sut/allow-read? false?)
-       (test-authorized_users-mismatch "amber" sut/allow-read? false?)
-       (test-authorized_groups-mismatch "amber" sut/allow-read? false?)
-       (test-authorized_users-match "amber" sut/allow-read? true?)
-       (test-authorized_groups-match "amber" sut/allow-read? true?))))
+   (fn [get-in-config]
+    (let [allow-read? #(sut/allow-read? %1 %2 get-in-config)]
+     (testing "amber TLPs should allow document read to same group"
+       (test-matching-user "amber" allow-read? true?)
+       (test-matching-group "amber" allow-read? true?)
+       (test-no-group "amber" allow-read? false?)
+       (test-user-group-mismatch "amber" allow-read? false?)
+       (test-authorized_users-mismatch "amber" allow-read? false?)
+       (test-authorized_groups-mismatch "amber" allow-read? false?)
+       (test-authorized_users-match "amber" allow-read? true?)
+       (test-authorized_groups-match "amber" allow-read? true?))))))
 
 (deftest allow-read?-tlp-red-max-record-visibility-group-test
   (with-max-record-visibility-group
-    #(testing "red TLPs should allow document read to owner only"
-       (test-matching-user "red" sut/allow-read? true?)
-       (test-matching-group "red" sut/allow-read? false?)
-       (test-no-group "red" sut/allow-read? false?)
-       (test-user-group-mismatch "red" sut/allow-read? false?)
-       (test-authorized_users-mismatch "red" sut/allow-read? false?)
-       (test-authorized_groups-mismatch "red" sut/allow-read? false?)
-       (test-authorized_users-match "red" sut/allow-read? true?)
-       (test-authorized_groups-match "red" sut/allow-read? true?))))
+   (fn [get-in-config]
+    (let [allow-read? #(sut/allow-read? %1 %2 get-in-config)]
```

Comment:
fn and let have 2-space indentation

Reply:
Indeed. A failed attempt at a compact diff, registers as one chunk with all the `sut/` deletions. Will fix.

Reply:
The weird indentation does give a nicer diff on my machine (just the `sut/`s are deleted), but it's useless on GitHub some reason.